payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherActivity.kt

```diff
@@ -39,10 +40,6 @@ internal class GooglePayLauncherActivity : AppCompatActivity() {
         GooglePayLauncherViewModel.Factory(args)
     }
 
-    private val errorReporter: ErrorReporter by lazy {
```

Comment:
I think as part of fixing the merge conflicts, the actual error reporting for some of these errors was removed. Can we fix that? Or if there are error cases that no longer exist, can we remove those cases from the ErrorReporter ErrorEvent enum?

Reply:
(could be fixed by sharing more code with GooglePayPaymentMethodLauncherActivity, see my comment there)

Reply:
Yeah, I thought I fixed these, but looks like I missed some. Good callout. I'll follow up.

Reply:
I didn't realize we had two classes that were so similar. So when I saw this missing after the rebase, I thought I fixed it. But I just added it to the other one 🤦.

I think it's probably worth having in both. And it's also probably worth consolidating these.